MobilliumDateFormatter/Classes/Date+DateComponentType.swift

```diff
@@ -5,14 +5,12 @@
 //  Created by Mehmet Salih Aslan on 18.02.2021.
 //
```

Comment:
We can add copyright to here
```suggestion
// Copyright © 2021 Mobillium. All rights reserved.
```